cmake/Modules/CodeCoverage.cmake

```diff
@@ -192,44 +191,48 @@ function(ADD_CODE_COVERAGE)
       set(OMIT_FLAGS --omit ${Coverage_EXCLUDES})
     endif()
 
+    add_custom_command(
+        OUTPUT ${PROJECT_BINARY_DIR}/${Coverage_NAME}_nosetests_python.xml
+        COMMAND ${PYTHON_COVERAGE_PATH} report --include "*${REAL_SOURCE_DIR}*" ${OMIT_FLAGS} || echo "WARNING: No python nosetests report to output"
+        COMMAND ${PYTHON_COVERAGE_PATH} xml  -o ${Coverage_NAME}_nosetests_python.xml --include "*${REAL_SOURCE_DIR}*" ${OMIT_FLAGS} || echo "WARNING: No python nosetests xml to output"
+        COMMAND ${CMAKE_COMMAND} -E make_directory ${PROJECT_BINARY_DIR}/python_nosetests_coverage/ || echo "WARNING: Error to create python nosetests coverage dir"
+        COMMAND mv ${PROJECT_BINARY_DIR}/.coverage* ${PROJECT_BINARY_DIR}/python_nosetests_coverage/ || echo "WARNING: No python nosetests report to move"
```

Comment:
ditto

Reply:
For this part, it seems `${CMAKE_COMMAND} -E move` does not support wildcard `*`, which makes command complicated.
I will change the other part.